From 782c2652920821fe6b7d5710911630b55f5efcec Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 4 Sep 2014 21:40:00 +0200
Subject: [PATCH] hibernate-resume: let's move all hibernate-resume tools into
 the same directory

They are closely related, so let's move them together, and clean up the
.c file naming while we are at it.
---
 Makefile.am                                                             | 2 +-
 .../hibernate-resume-generator.c}                                       | 0
 src/resume-generator/Makefile                                           | 1 -
 3 files changed, 1 insertion(+), 2 deletions(-)
 rename src/{resume-generator/resume-generator.c => hibernate-resume/hibernate-resume-generator.c} (100%)
 delete mode 120000 src/resume-generator/Makefile

diff --git a/Makefile.am b/Makefile.am
index e534a23886..9c946d7a92 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2168,7 +2168,7 @@ systemd_hibernate_resume_LDADD = \
         libsystemd-shared.la
 
 systemd_hibernate_resume_generator_SOURCES = \
-	src/resume-generator/resume-generator.c
+	src/hibernate-resume/hibernate-resume-generator.c
 
 systemd_hibernate_resume_generator_LDADD = \
 	libsystemd-label.la \
diff --git a/src/resume-generator/resume-generator.c b/src/hibernate-resume/hibernate-resume-generator.c
similarity index 100%
rename from src/resume-generator/resume-generator.c
rename to src/hibernate-resume/hibernate-resume-generator.c
diff --git a/src/resume-generator/Makefile b/src/resume-generator/Makefile
deleted file mode 120000
index d0b0e8e008..0000000000
--- a/src/resume-generator/Makefile
+++ /dev/null
@@ -1 +0,0 @@
-../Makefile
\ No newline at end of file
